src/components/_pages/PageCellar.tsx

```diff
@@ -131,22 +133,34 @@ const PageCellar: VFC<CellarPageProps> = ({ data: staticData }) => {
           <VStack spacing={6} align="flex-start">
             <BreadCrumb cellarName={name} />
             <HStack spacing={4}>
-              <Link href="/">
+              {/* <Link href="/">
```

Comment:
Can this be safely deleted, or is it commented out for a reason? If there's a reason, could we add that to the comment?